### PR TITLE
feat : filters and profile-page

### PR DESCRIPTION
**Pull request by mikesch803**
1. add filters to notes
2. add profile page
3. add logout

## files to check

- [ ] src/components/filter/Filter.jsx
- [ ] src/context/filter-context.jsx
- [ ] src/pages/profile/Profile.jsx
- [ ] src/reducer/filterReducer.jsx
- [ ] src/utils/functions/sortedNotes.js

### CODE REVIEW

**Comment by mikesch803** [comment]
##need to fix confict, so uploaded video preview [video link](https://drive.google.com/drive/folders/1z8UEPBuFWI449kKaIzkHfn-RqHF2vPCZ)

File: src/components/filter/Filter.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    <aside className="aside-filter">
+      <div className="filter-container m-b-1">
+        <h2 className="filter-title">
+          Filters
+          <button onClick={() => filterDispatch({ type: "CLEAR" })}>
+            CLEAR
+          </button>
+        </h2>
+        <h3>Sort by Date</h3>
+        <ul>
```
Comment (foolhardy21) [inline src/components/filter/Filter.jsx:17]: for this `ul`, you have not provided any `li` elements.

Comment (foolhardy21) [reply src/components/filter/Filter.jsx:17]: Similarly, other `ul` in this component too.

**Review by foolhardy21 — APPROVED** [review]
the code looks good. I've given one suggestion regarding the code.

**Review by hsnice16 — APPROVED** [review]
I think the other review has suggested all the changes that you can do. There is nothing new to suggest, just to do that. 
Good Job.

**Pull request merged**